src/test/java/com/coherent/task40/MultiselectTest.java

```diff
@@ -0,0 +1,50 @@
+package com.coherent.task40;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static com.coherent.task40.TestConstants.DROPDOWN_PAGE;
+
+public class MultiselectTest {
+    WebDriver driver = new ChromeDriver();
+    By multiSelectDropdown = By.id("multi-select");
+    List<String> actualOptionsList = new ArrayList<>();
+    List<String> expectedOptionsList = Arrays.asList("California", "Florida", "New Jersey");
+
+    @BeforeEach
+    void setUp() {
+        BaseTest.setup(driver, DROPDOWN_PAGE);
+    }
+
+    @Test
+    public void testDropdown() {
```

Comment:
The name of this method doesn't follow naming that you had in previous tests. 
Probably, better name could be dropdownTest()